website/docs/docs/building-a-dbt-project/metrics.md

```diff
@@ -9,30 +10,37 @@ id: "metrics"
 
 </Changelog>
 
+
 :::info Metrics are new
-An initial version of `metrics` was introduced in v1.0.0, following [vibrant community discusion](https://github.com/dbt-labs/dbt-core/issues/4071). Try them out, and let us know what you think!
+v1.0.0 includes an initial version of metrics, following a [vibrant community discussion](https://github.com/dbt-labs/dbt-core/issues/4071). Try them out, and let us know what you think!
 :::
 
 :::caution Metrics are experimental
-`metrics` will be released in v1.0, but they should _not_ be considered a stable API. We reserve the right to make breaking changes to their schema in future **minor** versions. We will aim for backwards compatibility whenever possible.
+v1.0 includes metrics, but they should be considered an _unstable_ API because they are experimental and subject to change. We reserve the right to make breaking changes to the metrics schema in future **minor** versions, but will aim for backwards compatibility when possible.
```

Comment:
* I used "unstable" instead of "not stable" because negatives can be tricky for non-native English speakers.
* I also only capitalized "Metrics" when beginning a sentence. It seems like it's a part of a feature and not a product feature on its own, right?